bciers/apps/registration/app/components/operations/registration/RegistrationSubmissionForm.tsx

```diff
@@ -46,43 +44,33 @@ const RegistrationSubmissionForm = ({
         }),
       },
     );
-    if (response?.error) {
-      setError(response.error);
-      setSubmitButtonDisabled(false);
-      setIsSubmitting(false);
-      return { error: response.error };
-    }
-    setIsSubmitted(true);
+    // errors are handled in MultiStepBase
+    return response;
   };
 
   return (
     <>
-      {isSubmitted ? (
-        <Success step={step} steps={steps} />
-      ) : (
-        <MultiStepBase
-          allowBackNavigation
-          baseUrl={`/register-an-operation/${operation}`}
-          baseUrlParams="title=Placeholder+Title"
-          cancelUrl="/"
-          formData={formState}
-          onSubmit={handleSubmit}
-          error={error}
-          schema={
-            isSubmitting
-              ? {
-                  title: "Submitting...",
-                  type: "object",
-                }
-              : schema
-          }
-          step={step}
-          steps={steps}
-          uiSchema={submissionUiSchema}
-          onChange={handleChange}
-          submitButtonDisabled={submitButtonDisabled}
-        />
-      )}
+      <MultiStepBase
+        allowBackNavigation
+        baseUrl={`/register-an-operation/${operation}`}
+        cancelUrl="/"
+        formData={formState}
+        onSubmit={handleSubmit}
+        schema={
+          isSubmitting
```

Comment:
I'm considering moving the `isSubmitting` logic to MultiStepBase. Some forms have it at the moment and some don't. Thoughts?